spec.html

```diff
@@ -32027,79 +32027,65 @@ <h1>Time Values and Time Range</h1>
+        <p>Months are identified by an integer in the range 0 to 11, inclusive. A month value of 0 specifies January; 1 specifies February; 2 specifies March; 3 specifies April; 4 specifies May; 5 specifies June; 6 specifies July; 7 specifies August; 8 specifies September; 9 specifies October; 10 specifies November; and 11 specifies December.</p>
+        <p>The position of a month _m_ within a year depends upon whether or not the year is a leap year:</p>
+        <emu-eqn id="eqn-DaysWithinYearBeforeMonth" aoid="DaysWithinYearBeforeMonth">DaysWithinYearBeforeMonth(_m_, _leapDaysInYear_)
+          = *+0*<sub>𝔽</sub> if _m_ = 0
+          = *31*<sub>𝔽</sub> if _m_ = 1
+          = *59*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 2
+          = *90*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 3
+          = *120*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 4
+          = *151*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 5
+          = *181*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 6
+          = *212*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 7
+          = *243*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 8
+          = *273*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 9
+          = *304*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 10
+          = *334*<sub>𝔽</sub> + _leapDaysInYear_ if _m_ = 11
         </emu-eqn>
+        <emu-clause id="sec-MonthFromTime" aoid="MonthFromTime" oldids="eqn-MonthFromTime">
+          <h1>MonthFromTime ( _t_ )</h1>
+          <p>The abstract operation MonthFromTime maps a time value _t_ to a month number. It performs the following steps:</p>
+          <emu-alg>
+            1. Let _day_ be DayWithinYear(_t_).
+            1. Let _leapDaysInYear_ be LeapDaysInYear(YearFromTime(_t_)).
+            1. Let _month_ be *+0*<sub>𝔽</sub>.
+            1. Repeat, while _month_ &lt; *11*<sub>𝔽</sub>,
+              1. If _day_ &lt; DaysWithinYearBeforeMonth(_month_ + 1, _leapDaysInYear_), return _month_.
+              1. Set _month_ to _month_ + *1*<sub>𝔽</sub>.
+            1. Return *11*<sub>𝔽</sub>.
+          </emu-alg>
+        </emu-clause>
         <p>where</p>
         <emu-eqn id="eqn-DayWithinYear" aoid="DayWithinYear">DayWithinYear(_t_) = Day(_t_) - DayFromYear(YearFromTime(_t_))</emu-eqn>
-        <p>A month value of *+0*<sub>𝔽</sub> specifies January; *1*<sub>𝔽</sub> specifies February; *2*<sub>𝔽</sub> specifies March; *3*<sub>𝔽</sub> specifies April; *4*<sub>𝔽</sub> specifies May; *5*<sub>𝔽</sub> specifies June; *6*<sub>𝔽</sub> specifies July; *7*<sub>𝔽</sub> specifies August; *8*<sub>𝔽</sub> specifies September; *9*<sub>𝔽</sub> specifies October; *10*<sub>𝔽</sub> specifies November; and *11*<sub>𝔽</sub> specifies December. Note that <emu-eqn>MonthFromTime(*+0*<sub>𝔽</sub>) = *+0*<sub>𝔽</sub></emu-eqn>, corresponding to Thursday, 1 January 1970.</p>
+        <p>Note that <emu-eqn>MonthFromTime(0) = 0</emu-eqn>, corresponding to Thursday, 01 January, 1970.</p>
```

Comment:
Double-check the types here — I think this might be `MonthFromTime(*+0*<sub>𝔽</sub>) = 0` (also taking into account the above comment about the return type of MonthFromTime)